client/my-sites/upgrades/domain-management/domain-connect/constants.js

```diff
@@ -0,0 +1,10 @@
+export const actionType = {
```

Comment:
Kudos on using constants - I wish this was a more common approach, instead of duplicating magic strings all over the place (which are just a bit better than magic numbers).
Tip: you can use [something like `keyMirror`](https://github.com/Automattic/wp-calypso/blob/ea60bf27851b4371dc7c1bed43e21c8d874bf60d/client/lib/domains/constants.js#L6-L11), if you don't care about the value of the keys:
```
const actionType = keyMirror( {
      CLOSE: null,
      READY_TO_SUBMIT: null,
      SUBMITTING: null,
  } );
```